Jenkinsfile

```diff
@@ -58,6 +122,20 @@ pipeline {
                 }
 			}
 		}
+		stage ('Bump Development Version') {
+			when {
+				expression {
+					return BRANCH_NAME == '1.3.x' && params.RELEASE
+				}
+			}
+			steps {
+				script {
+					sh "mvn versions:set -DnewVersion=${NEW_VERSION}-SNAPSHOT --batch-mode"
+                    sh "git commit --all --message 'New Development Version $NEW_VERSION-SNAPSHOT'"
```

Comment:
Indentation does go a bit wonky in this file.

Reply:
Indeed it does - will do a separate whitespace commit